pv_site_api/enode_auth.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import httpx
+
+
+class EnodeAuth(httpx.Auth):
+    def __init__(self, access_token):
+        self.access_token = access_token
```

Comment:
Add a type hint and make this optional, since we'll start with no access token as mention right?
```suggestion
    def __init__(self, access_token: Optional[str] = None):
        self._access_token = access_token
```